README.md

```diff
@@ -40,6 +40,9 @@ To get started, you have to install the dependencies (mainly snowpack)
 npm install
 ```
 
+### Environment Labels
+In order to run locally you will need to retrieve the `.env` file from here.........location pending
```

Comment:
This is a very good point. If nothing else, I'd like to see a `.env.example` file with the keys that we're expecting, even with dummy values. We'll have to figure out how to share these "secrets" with the whole team